src/components/collapsible/styles/collapsibleContainer.scss

```diff
@@ -0,0 +1,14 @@
+$header-height: 50px;
+$content-padding: 20px 40px;
```

Comment:
You could add these in vapor and import them here so they can be used elsewhere. I think content padding already exists too.

Reply:
agree with @sroy3 this is too generic to be at the component level. You have one of them already in vapor but I don't think content-padding exists in vapor, header-height is `$collapsible-height`. To use the variables in vapor you can do `@import '~coveo-styleguide/scss/variables.scss';` at the top of the file and they will be available to you afterward.

Reply:
you can look at BorderedLine.tsx for an example